test/plugin/.testConfig.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
      */
     projectTestConfig: {
         mocha: {
-            timeout: 30000,
+            timeout: 60000,
```

Comment:
restore this, I found out there were some tests failing due to timeouts as well. However, we need to see where this issue is coming from. By incrementing the timeout we are just masking the issue.